src/game/hooks/Info/PlayerJoinLeave.cpp

```diff
@@ -1,15 +1,53 @@
+#include "core/commands/BoolCommand.hpp"
+#include "core/frontend/Notifications.hpp"
 #include "core/hooking/DetourHook.hpp"
+#include "game/features/Features.hpp"
 #include "game/hooks/Hooks.hpp"
-#include "core/frontend/Notifications.hpp"
 #include "game/rdr/Natives.hpp"
 #include "network/CNetGamePlayer.hpp"
 
+
+namespace YimMenu
+{
+	std::string RemoveColorFromName(std::string name)
```

Comment:
I don't really know why you are removing these characters from their name. And this does not look like the place for an utility function

Reply:
Yeah, to be honest I don't really know either but my thought process was if they are spoofing their name we might aswell remove the color to get their "real" name( that could very well be spoofed also )